config/200-channelable-clusterrole.yaml

```diff
@@ -0,0 +1,25 @@
+# Copyright 2019 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Use this aggregated ClusterRole when you need read/write access to "Channelables"
```

Comment:
The permissions given do not include create or delete, so I would rephrase this to:

```suggestion
# Use this aggregated ClusterRole when you need read and update permissions on "Channelables".
```

Reply:
Done!